CHANGELOG.md

```diff
@@ -1,5 +1,68 @@
 # Version changelog
 
+## Release v0.28.0
```

Comment:
nit: Just for consistency, in the past releases, we don't mention `v` in the title of release in changelog. 

v0.28.0 -> 0.28.0